android/build/mfx_defs_internal.mk

```diff
@@ -0,0 +1,34 @@
+# Purpose:
+#   Defines include paths, compilation flags, etc. to build Media SDK
+# internal targets (libraries, test applications, etc.).
+#
+# Defined variables:
+#   MFX_CFLAGS_INTERNAL - all flags needed to build MFX targets
+#   MFX_CFLAGS_INTERNAL_HW - all flags needed to build MFX HW targets
+#   MFX_C_INCLUDES_INTERNAL - all include paths needed to build MFX targets
+#   MFX_C_INCLUDES_INTERNAL_HW - all include paths needed to build MFX HW targets
+
+MFX_CFLAGS_INTERNAL := \
+    $(MFX_CFLAGS) \
+    $(MFX_CFLAGS_STL)
+
+MFX_CFLAGS_INTERNAL_32 := -DLINUX32
+MFX_CFLAGS_INTERNAL_64 := -DLINUX32 -DLINUX64
+
+MFX_CFLAGS_INTERNAL_HW := \
+    $(MFX_CFLAGS_INTERNAL) \
+    -Wno-error=non-virtual-dtor
```

Comment:
-Wno-error=non-virtual-dtor still presents. If build is ok without it, need to remove.

Reply:
Build fails without this option

Reply:
Error look valid. Instead of adding this warning, need to add virtual destructors in the appropriate classes!

Reply:
The error occurs in CM headers, but we cannot modify these classes as they used in API and we can get wrong input. To minimize potential risk we can keep the warning for common code, but disable it for particular files

Reply:
Ok, let's try to disable warning for particular files.

Reply:
done